CleverPush/CleverPush/Source/CleverPushInstance.m

```diff
@@ -676,6 +676,17 @@ - (void)setMaximumNotificationCount:(int)limit {
     [userDefaults synchronize];
 }
 
+#pragma mark - getDeviceToken.
+- (void)getDeviceToken:(void(^)(NSString *))callback {
+    if (deviceToken) {
```

Comment:
When the customer calls this method before deviceToken is defined, it will not work.

Please implement something like we have for the subscriptionId (pendingSubscriptionListeners):
https://github.com/cleverpush/cleverpush-ios-sdk/blob/92859a3deea164f03fb33b0e636d683007bbfb35/CleverPush/CleverPush/Source/CleverPushInstance.m#L695

Once we get the deviceToken here:
https://github.com/cleverpush/cleverpush-ios-sdk/blob/92859a3deea164f03fb33b0e636d683007bbfb35/CleverPush/CleverPush/Source/CleverPushInstance.m#L1102
You can call the pendingDeviceTokenListeners:
https://github.com/cleverpush/cleverpush-ios-sdk/blob/92859a3deea164f03fb33b0e636d683007bbfb35/CleverPush/CleverPush/Source/CleverPushInstance.m#L1265